tests/robottelo/test_func_locker.py

```diff
@@ -0,0 +1,321 @@
+# coding: utf-8
+
+from __future__ import with_statement
```

Comment:
As we support python 2.7+, `with` should be there. Or python 3 has some differences in `with` statement implementation which i'm not aware of?